7_react/react1/week1/vite-project/src/components/ItemListDisplay..jsx

```diff
@@ -0,0 +1,9 @@
+const ListingItems = (props) => {
```

Comment:
I would suggest to have the same name for component and file where component is located, to avoid confusion. For example it can be `const = ItemListdisplay` the same way as file named.